src/base/search/searchhandler.h

```diff
@@ -87,5 +87,5 @@ class SearchHandler : public QObject
     QTimer *m_searchTimeout;
     QByteArray m_searchResultLineTruncated;
     bool m_searchCancelled = false;
-    QList<SearchResult> m_results;
+    QVector<SearchResult> m_results;
```

Comment:
Are you sure about this one? It fills up in a few iterations, so you can't guess its size to avoid memory reallocation.

Reply:
reverted.

Reply:
>reverted

Really I meant only this field and its getter. It seems you can use QVector in other cases where you used it initially.

Reply:
oops you're right, changed.